pipeline/src/lib.rs

```diff
@@ -69,13 +92,31 @@ pub async fn session_from_config<F: ConclusionFeed + 'static>(
     // Set the path within the bucket for the doc_state table
     const DOC_STATE_OBJECT_STORE_PATH: &str = "/ceramic/v0/doc_state/";
     url.set_path(DOC_STATE_OBJECT_STORE_PATH);
-    ctx.register_listing_table(
+    // Register doc_state_persistent as a listing table
+    ctx.register_table(
+        "_internal.doc_state_persistent",
```

Comment:
it seems like we have a lot of strings for table names. would it make sense to add a few `const`s so they're defined in one place?